compiler/options.nim

```diff
@@ -1027,6 +1027,8 @@ proc parseIdeCmd*(s: string): IdeCmd =
   of "recompile": ideRecompile
   of "changed": ideChanged
   of "type": ideType
+  of "declaration": ideDeclaration
+  of "expand": ideExpand
   else: ideNone
```

Comment:
I saw that the compiler has it's own code for converting strings to ideCmd types.
nimsuggest duplicated this code and added a few extra commands. To reduce the pointless copying I just added the missing commands to the compilers parser.